rpcserver.go

```diff
@@ -7713,3 +7713,9 @@ func (r *rpcServer) getInboundPolicy(ctx context.Context, chanID uint64,
 
 	return policy, nil
 }
+// DecodeAssetPayReq decodes an incoming invoice, then uses the RFQ system to
+// map the BTC amount to the amount of asset units for the specified asset ID.
+func (r *rpcServer) DecodeAssetPayReq(ctx context.Context,
+	payReq *tchrpc.AssetPayReqString) (*tchrpc.AssetPayReq, error) {
+	return &resp, nil
+}
```

Comment:
I don't think this empty placeholder method is necassary in this commit. It could be moved to the last commit along with the rest of the method implementation: `rpc: implement DecodeAssetInvoice`.

Reply:
I added this here so the commit would still build, but I realize now I forgot to add the `var` declaration -- adding it now.

Reply:
Ahh I forgot we have `UnimplementedTaprootAssetChannelsServer` -- ok can remove it.